docs/appendix.rst

```diff
@@ -342,6 +342,24 @@ This documentation uses a few special terms to refer to Python types:
       can be persisted in a pywbem connection file and then referenced by
       name in future pywbemcli calls.
 
+   tab-completion
+      Also known as comand-line completion or shell completion, is a mode
+      where the terminal processor fils in partially thyped commands when the
+      user hits <TAB> or <TAB><TAB>. This mode of operation is available with
+      pywbemcli when a terminal shell that enables tab-completion is being
+      used (ex. bash, zsh) and when tab-completion has been activated for
+      pywbemcli.
+
+   auto-suggestion
+      Auto-suggestion is a way to propose input on the command line based on
+      the history of previous commands. The input is compared to
+      the history and when there is an entry in the
+      history file with the given text the proposed completion is show as gray
+      text behind the current input. Pressing the right arrow → or c-e will
```

Comment:
I suggest to be consistent with how we refer to special keys such as tab, ctrl, alt. The current approach for `<TAB>` looks good to me, ie. 
`<CTRL>` and `<ALT>`

Reply:
Reviewed and found two with <tab>. fixed.  Also generalized to the separate words tab completion now all have hyphen for consistency
DONE